delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -180,6 +180,20 @@ final class ResourcesRoutes(
                           }
                         }
                       },
+                      (pathPrefix("validate") & get & pathEndOrSingleSlash & idSegmentRef(id)) { id =>
+                        operationName(s"$prefixSegment/resources/{org}/{project}/{schema}/{id}/validate") {
+                          authorizeFor(ref, Read).apply {
+                            emit(
+                              OK,
+                              resources
+                                .validate(id, ref, schemaOpt)
+                                .map(_.map(_.json))
```

Comment:
It would be nice to return Json-LD and not json
So if you return  a `ValidationResult`, you will need a `JsonLdEncoder` there.